assets/src/styles/blocks/ShareButtons/ShareButtonsStyles.scss

```diff
@@ -0,0 +1,50 @@
+@mixin sharedStyles($color) {
+  background: $color;
+
+  &:hover,
```

Comment:
We should never use the same styles for hover, focus and active, these need to be distinct for a11y.

In this case you can definitely leave out the `focus` state, these should get a focus border from our general styles.

Reply:
This is how is currently implemented
https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/components/_share-buttons.scss#L25-L62

Reply:
Ah I didn't realize these styles were copied over. Why do we even need to copy them? Can't they use the existing styles?

Reply:
In which case of course we need to fix this issue in the existing styles :smile:

Reply:
Don't you think it's better if we differentiate the styles from the block?

Reply:
Do we have a specification somewhere saying we want a different style of share buttons for this block?

Reply:
I assume that if we don't have a specification, it makes most sense to use the existing style. If we find in the future we want a different style here, we can at that point add separate CSS.

Reply:
Ok! I'm going to use the current sharbuttons styles, so, should I remove `active` from [here](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/components/_share-buttons.scss#L25-L62), shouldn't I?

Reply:
You can also remove `focus` there, as it already has a focus outline from the general button styles.